pkg/commands/commands.go

```diff
@@ -42,9 +41,9 @@ See https://github.com/kubernetes-sigs/kustomize
 	}
 
 	c.AddCommand(
-		newCmdBuild(stdOut, stdErr, fsys),
+		newCmdBuild(stdOut, fsys),
 		newCmdDiff(stdOut, stdErr, fsys),
-		newCmdEdit(stdOut, stdErr, fsys),
+		newCmdEdit(fsys),
```

Comment:
This change will leave us inconsistent format for `newCmd*` functions, which is not very nice. Can you add a `TODO` here for a later fix?

Reply:
I've added TODO, pushed new PR.